style.css

```diff
@@ -1 +1,100 @@
+body {
+  width: 1200px;
+}
 
+header {
+  width: 900px;
+  height: 50px;
+  margin: 0px auto;
+  margin-left: 300px;
```

Comment:
This `margin-left` is destroying your centering. It pushes everything far to the right.